logging/configure.go

```diff
@@ -0,0 +1,105 @@
+package logging
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/armory-io/monitoring/log/formatters"
+	"github.com/armory-io/monitoring/log/hooks"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	Remote RemoteLoggingConfig `json:"remote" yaml:"remote"`
+	JSON   FormatJson          `json:"json" yaml:"json"`
```

Comment:
Can we also customize non JSON output? I don't know if it's useful but many logging libraries allow you to have different "appenders" with different format. e.g. send to console in human readable format, send JSON to remote location, etc. I have not had the need but perhaps we could either use the same concept of appenders. e.g.

```
logging:
  remote:
     ...
  appenders:
    default:
      destination: stdout
      level: DEBUG
      format: json
      fields: [...]
    myfile:
      destination: /var/log/myservice.log
      format: text
      fields: [...]
```
The use of `destination` is definitely more limited in the container world. `level` seems required though.